### PR TITLE
[15.0][MIG] rma: Migration to 15.0

### PR DESCRIPTION
**Pull request by ghost**
with the changes from the review

### CODE REVIEW

**Review by victoralmau — CHANGES_REQUESTED** [review]

File: rma/tests/test_rma.py [hunk 1]
```diff
@@ -0,0 +1,803 @@
...
+        )
+        cls.partner_shipping = cls.res_partner.create(
+            {
+                "name": "Partner shipping test",
+                "parent_id": cls.partner.id,
+                "type": "delivery",
+            }
+        )
+        cls.finalization_reason_1 = cls.env["rma.finalization"].create(
+            {"name": ("[Test] It can't be repaired and customer doesn't want it")}
```
Comment (victoralmau) [inline rma/tests/test_rma.py:66]:
```suggestion
            {"name": "[Test] It can't be repaired and customer doesn't want it"}
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Review by chienandalu — COMMENTED** [review]
Thanks @nikolausweingartmair please keep an eye on @victoralmau proposed changes as they're quite relevant: https://github.com/OCA/rma/pull/290/commits/2b4bcd5c4806bb9da60587e108a83c2661368474#diff-ed3cac0ed4c2bb088667ed1827b89e117ff2f2acc5fbe6c05eb16c2fe1eabcd8

**Comment by ghost** [comment]
I incorporated the changes from 2b4bcd5.

**Review by victoralmau — APPROVED** [review]
Good job!

**Pull request merged**